utils/validate_images.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+const core = require('@actions/core');
+const isImage = require('is-image');
+const {
+  getImageCount,
+  getFileSize,
+  getFileExtension,
+  globFiles,
+  isDirectory,
+} = require('./helpers');
+
+const BASE_PATH = '../packs/';
+const MAX_SIZE = 4000000;
+const MAX_NUM_IMG = 6;
+const ALLOWED_IMG_EXT = ['.png', '.jpeg', '.jpg', '.svg'];
+
+/**
+ * Validate all folders contain no more than MAX_NUM_IMG images
```

Comment:
this is something you dont need to address, and just my opinion. you might describe the side effect that happens when you call the function & validation fails in the description / summary section of the doc.

so for this, and the other functions, i might write - 'Validate all folders contain no more than MAX_NUM_IMG images. if validation fails, process.exitCode is set to 1.'